### DISCUSSION TITLE
Are Github Octernships Discontinued?

### DISCUSSION THREAD

**Discussion by SkySingh04** [community/community#122799] [GitHub Education]
### Hi everyone,

Hey Github Community, Are Github octernships discontinued? I could not find the application link/tab on my Github Education page. Can someone provide updates regarding this?

I fulfill all of the criteria for the above, unless there is some specific requirement I am not updated about?

![image](https://github.com/community/community/assets/114267538/1146223e-0232-43fd-a530-d4c7a44288fc)

**Comment by manthan0x23** [comment]
Yeah same here , : (
